src/api/app/models/package.rb

```diff
@@ -397,7 +397,12 @@ def unlock_by_request(request)
 
   def sources_changed(opts = {})
     dir_xml = opts[:dir_xml]
-    update_activity
+
+    # to call update_activity before filter
+    # NOTE: We need `Time.now`, otherwise the old tests suite doesn't work,
+    # remove it when removing the tests
+    update({updated_at: Time.now})
```

Comment:
Why is a behaviour change okay with the new test suite?

I mean this makes a differentce on production, no?